Source/BugsnagNotifier.m

```diff
@@ -287,6 +290,21 @@ - (void) start {
 #endif
 }
 
+- (void)setupConnectivityListener {
+    NSString *hostUrl = [self apiHostUrl];
+    self.apiReachable = [BSGReachability reachabilityWithHostName:hostUrl];
```

Comment:
`hostUrl` is URL, not a hostname. It definitely works 🤔, but I'd be concerned that if the implementation of `SCNetworkReachabilityCreateWithName` ever changes, this code would break unexpectedly.